code/modules/bitrunning/server/map_handling.dm

```diff
@@ -57,7 +57,7 @@
 	is_ready = TRUE
 
 	var/spawn_chance = clamp((threat * glitch_chance), 5, threat_prob_max)
-	if(prob(spawn_chance))
+	if(!no_glitches && prob(spawn_chance))
```

Comment:
Double negatives are annoying, change the var from `no_glitches = FALSE` to just `spawn_glitches = TRUE`